projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -396,4 +405,12 @@ export class NavigationUIComponent implements OnInit, OnDestroy {
     }
     return depth > 0 && !node?.children ? -1 : 0;
   }
+
+  ariaLabel(node: NavigationNode) {
+    return !this.isHeadersLinksSlotNavigation ? node.title : undefined;
```

Comment:
Maybe we should include a test to ensure this behaviour works as expected? This could help avoid breaking this feature again and additionally document it for others